### PR TITLE
Update README.md for old and new functions

### PR DESCRIPTION
**Pull request by susgo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,41 @@ $csv->auto('data.csv');
 print_r($csv->data);
 ```
 
+**Parse data with offset**
+* ignoring the first X (e.g. two) number of rows
```
Comment (Fonata) [inline README.md]: This is a little bit hard to read. From my perspective, removing the words "number of" fixes the sentence.

File: README.md [hunk 2]
```diff
@@ -77,6 +77,41 @@ $csv->auto('data.csv');
...
+**Limit the number of returned data rows**
+```php
+$csv = new ParseCsv\Csv();
+$csv->limit = 5;
+$csv->parse('data.csv');
+print_r($csv->data);
+```
+
+**Get total number of data rows without parsing whole data**
+* exclusive heading line if present
```
Comment (Fonata) [inline README.md]: - "Excluding"
- "(see `$csv->header` property)"

File: README.md [hunk 3]
```diff
@@ -77,6 +77,41 @@ $csv->auto('data.csv');
...
+```php
+$csv = new ParseCsv\Csv();
+$csv->load_data('data.csv');
+$count = $csv->getTotalRowCount();
+print_r($count);
+```
+
+**Get data type foreach Column**
+* Check data type for each column and returns the most commonly.
```
Comment (Fonata) [inline README.md]: Let's combine the 2 lines into 1: "**Get most common data type for each column**"

(the -ly version needs to be followed by a verb)

**Comment by susgo** [comment]
@Fonata All requested changes done.

**Pull request merged**